pkg/test/framework/resource/settings.go

```diff
@@ -195,6 +197,13 @@ func (s *Settings) SkipVMs() {
 	s.SkipWorkloadClasses = append(s.SkipWorkloadClasses, "vm")
 }
 
+// Checks if dual-stack mode is to be enabled for tests based on the IpFamilies
+func (s *Settings) SetDualStack() {
```

Comment:
I would just hoist the body of this function into the caller, or rename it `ParseIPFamilies` - `SetDualStack` is confusing.